### DISCUSSION TITLE
How to add a custom layout for some docs

### DISCUSSION THREAD

**Discussion by hustcer** [vuepress/core#960] [Q&A] [answered]
Hi, 

I'm one of the maintainer of www.nushell.sh, and I have just upgraded the doc site from `vuepress` to `vuepress-next`, everything is okay except that I have tried to add a custom layout for all docs in `book/commands`, however, it doesn't work. So I have to insert a  script tag in the docs which is ugly, for example: https://github.com/nushell/nushell.github.io/blob/main/book/commands/alias.md. Before upgrading to `vuepress-next` we can use custom `command` layout to make the doc to be clean and tidy, like: https://raw.githubusercontent.com/nushell/nushell.github.io/cf006aaf8849b2b2e74546fcf2152cedc8ebcdde/book/commands/alias.md. My question is how can I do that in `vuepress-next`?  Thanks

**Comment by meteorlxy** [comment, marked as answer]
Hi,  You can still [use a custom layout](https://v2.vuepress.vuejs.org/reference/frontmatter.html#layout). So i'm not quite sure what your problem is. Could your please provide more context?

BTW, the `$frontmatter` is still available (though not documented yet 😅 )

**Reply by hustcer** [reply]
Glad to know `$frontmatter` works, I tried before but failed (maybe caused by other errors), and It does work now. Thanks
